### DISCUSSION TITLE
I can't find a direct correspond between the readme.md and auth.json

### DISCUSSION THREAD

**Discussion by t3hduck** [UltimaHoarder/UltimaScraper#1268] [Q&A]
Hi
I'd like to test the scraper but the auth.json on readme.md is different from the auth.json created by the app.
for example:
```
    "auth_hash": "",
    "auth_uniq_": "",
```
Another question:
what is the different between username and email/password?
Can I have some update on how to fill correct info in auth.json?
thank you in advance

**Comment by UltimaHoarder** [comment]
Just download the cookie helper in the faq

Email and password will launch a browser that will automatically configure it for you
